tsconfig.json

```
@@ -3,10 +3,12 @@
   "compilerOptions": {
     "module": "esnext",
     "lib": ["dom", "esnext"],
+    "target": "es2016",
```

```suggestion
    "target": "es2019",
```

Shouldn't this be `es2019` for node 12?

---

:thinking: I don't think there are any ES2019 features being used in the codebase.
But it probably wouldn't hurt to target `es2019` either :man_shrugging: